content/working_with/console/pages/agents-page.md

```diff
@@ -0,0 +1,14 @@
+---
+layout: bt_wiki
+title: Agents Page
+category: Console
+draft: false
+weight: 160

```

Comment:
**Agents Page** and **Sites Page** should be swapped to reflect the actual order in pages list in the application, so I think that `weight` should be changed either here or in `site-management-page.md`.

Reply:
Done